frontend/src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,97 @@
+@import "../../variable.scss";
+
+.navbar_list {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  background: $gradient90deg;
+  height: 4rem;
+  width: 18rem;
+  border-radius: 24px;
+  border: $borderDark;
+  box-shadow: $boxShadow;
+  position: fixed;
+  bottom: 5%;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.navbar_element {
+  list-style: none;
+  width: 33.33%;
+  height: 100%;
+}
+.navbar_element button {
+  all: unset;
+  cursor: pointer;
+}
+
+.navbar_element a,
+.navbar_element button {
+  width: 100%;
```

Comment:
Idem

Reply:
s100% needed for these elements, other units won't work here